notes/PRODUCT.md

```diff
@@ -0,0 +1,32 @@
+# Product Thinking
+The art to defining and solving a problem with a specified target audience, strategy, and goal in mind.
+
+## Layout
+In order to ______ (vision) <br>
+our product will solve ________ (target audience) <br>
+problem of _________ (user problem) <br>
+by giving them _________ (strategy).  <br>
+We know if our product works when we see a _________ (goal). <br>
+
+### User First
```

Comment:
I think these`###`-level sections demand some kind of introductory. Also these are lifted directly from the articles you posted below, so attribution would be important, e.g.

`This section adpated from [Article title](url)`

Reply:
Did some searching, I found the following verbiage:
This section adapted from Reference 8.